testsuite/features/step_definitions/common_steps.rb

```diff
@@ -1346,3 +1346,14 @@ def server_secret
   log "Reactivation Key: #{react_key}"
   step %(I enter "#{react_key}" as "reactivationKey")
 end
+
+When(/^I update ReportDB with Taskomatic$/) do
```

Comment:
Maybe better don't use the word Taskomatic, and do something like this?

```suggestion
When(/^I schedule a task to update reporting$/) do
```